scripts/.phantom-run-tests.js

```diff
@@ -5,42 +5,42 @@ var loggingWorks = false;
 var someTestsFailed = false;
 
 function installConsoleListener() {
-  page.onConsoleMessage = function(msg) {
-    if (msg.substring(0, logToken.length) == logToken) {
-      var commandString = msg.substring(logToken.length);
-      if (commandString == 'started') {
-        loggingWorks = true;
-      } else if(commandString == 'fail') {
-        someTestsFailed = true;
-      }
-    } else {
-      console.log(msg);
-    }
-  };
+    page.onConsoleMessage = function(msg) {
```

Comment:
I suggest we stick with two spaces. MDN recommends it too [1].
[1] https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style